remote/mdb_service_scripting.go

```diff
@@ -5,6 +5,11 @@ import (
 	"io"
 
 	"github.com/evergreen-ci/mrpc/mongowire"
+	"github.com/evergreen-ci/mrpc/shell"
+	"github.com/mongodb/jasper/options"
+	"github.com/mongodb/jasper/scripting"
+	"github.com/pkg/errors"
+	"gopkg.in/mgo.v2/bson"
```

Comment:
we shouldn't introduce new users of the legacy library. 

I'd vaguely prefer if things used birch, but if we need reflection (and/or existing parts of the service uses reflection), then we can/should use the new library